src/group_cmd.cpp

```diff
@@ -724,7 +724,7 @@ CommandCost CmdSetGroupFlag(DoCommandFlag flags, GroupID group_id, GroupFlags fl
 	Group *g = Group::GetIfValid(group_id);
 	if (g == nullptr || g->owner != _current_company) return CMD_ERROR;
 
-	if (flag >= GroupFlags::GF_END) return CMD_ERROR;
+	if (flag != GroupFlags::ReplaceProtection && flag != GroupFlags::ReplaceWagonRemoval) return CMD_ERROR;
```

Comment:
Can both flags be set at once? If so, that would cause an error. (And if not, these don't need to be bit flags at all.)

Reply:
They can definitely both be set on the group, but with the command only one could (and can) be set at a time.

But this does subtly alter the command parameter indeed. Not sure if that's a problem.

Basically `enum class` does not work with all the HasBit stuff. Maybe that's something to fix instead.